### PR TITLE
Modified the previous validtion function for checking domain protocol

### PR DESCRIPTION
**Pull request by prio101**
### Here is change:
- I modified the `stripProtocol()` function with `preg_replace()` and modified the `const` values.

I am looking forward some feedback .

### CODE REVIEW

File: src/Http/Validation/Domain.php [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
 class Domain implements Validator
 {
-    const PATTERN_STRIP_PROTOCOL = '/:\d*$/';
+    const PATTERN_STRIP_PORT = '/:\d*$/';
+    const PATTERN_STRIIP_PROTOCOL = '/[w]*:[\/]*/' ;
```
Comment (jasonlewis) [inline src/Http/Validation/Domain.php]: Got an extra "I" in "STRIP".

File: src/Http/Validation/Domain.php [hunk 2]
```diff
@@ -50,8 +51,9 @@ public function validate(Request $request)
      */
     protected function stripProtocol($domain)
     {
-        if (Str::contains($domain, '://')) {
-            $domain = substr($domain, strpos($domain, '://') + 3);
+        
+        if ($domainStrippedProtocol = preg_replace(self::PATTERN_STRIP_PROTOCOL, null, $domain)){
```
Comment (jasonlewis) [inline src/Http/Validation/Domain.php]: Can just be `$domainStripped`.

**Comment by jasonlewis** [comment]
Aside from the commit comments. What's the benefit here?

**Comment by hskrasek** [comment]
@prio101 Like @jasonlewis said, what exactly does this PR aim to do?

**Comment by thilanga** [comment]
Closing this as no feedbacks from  @prio101

**Pull request closed without merging**